mappings/net/minecraft/item/ItemType.mapping

```diff
@@ -1,56 +1,122 @@
+	FIELD S bread Lgm;
+	FIELD T helmetCloth Lgm;
+	FIELD U chestplateCloth Lgm;
+	FIELD V leggingsCloth Lgm;
+	FIELD W bootsCloth Lgm;
+	FIELD X helmetChain Lgm;
+	FIELD Y chestplateChain Lgm;
+	FIELD Z leggingsChain Lgm;
+	FIELD aA redstone Lgm;
+	FIELD aB snowball Lgm;
+	FIELD aC boat Lgm;
+	FIELD aD leather Lgm;
+	FIELD aE milk Lgm;
+	FIELD aF brick Lgm;
+	FIELD aG clay Lgm;
+	FIELD aH reeds Lgm;
+	FIELD aI paper Lgm;
+	FIELD aJ book Lgm;
+	FIELD aK slimeball Lgm;
+	FIELD aL minecartChest Lgm;
+	FIELD aM minecartFurnace Lgm;
+	FIELD aN egg Lgm;
+	FIELD aO compass Lgm;
+	FIELD aP fishingRod Lgm;
+	FIELD aQ clock Lgm;
+	FIELD aR yellowDust Lgm;
+	FIELD aS fishRaw Lgm;
+	FIELD aT fishCooked Lgm;
+	FIELD aU dyePowder Lgm;
+	FIELD aV bone Lgm;
+	FIELD aW sugar Lgm;
+	FIELD aX cake Lgm;
+	FIELD aY bed Lgm;
+	FIELD aZ diode Lgm;
+	FIELD aa bootsChain Lgm;
+	FIELD ab helmetIron Lgm;
+	FIELD ac chestplateIron Lgm;
+	FIELD ad leggingsIron Lgm;
+	FIELD ae bootsIron Lgm;
+	FIELD af helmetDiamond Lgm;
+	FIELD ag chestplateDiamond Lgm;
+	FIELD ah leggingsDiamond Lgm;
+	FIELD ai bootsDiamond Lgm;
+	FIELD aj helmetGold Lgm;
+	FIELD ak chestplateGold Lgm;
+	FIELD al leggingsGold Lgm;
+	FIELD am bootsGold Lgm;
+	FIELD an flint Lgm;
+	FIELD ao porkchopRaw Lgm;
+	FIELD ap porkchopCooked Lgm;
+	FIELD aq painting Lgm;
+	FIELD ar appleGold Lgm;
+	FIELD as sign Lgm;
+	FIELD at doorWood Lgm;
+	FIELD au bucket Lgm;
+	FIELD av bucketWater Lgm;
+	FIELD aw bucketLava Lgm;
+	FIELD ax minecart Lgm;
+	FIELD ay saddle Lgm;
+	FIELD az dorrIron Lgm;
 	FIELD b rand Ljava/util/Random;
-	FIELD ba COOKIE Lgm;
-	FIELD bb MAP Lwr;
-	FIELD bc SHEARS Lbl;
-	FIELD bd RECORD_13 Lgm;
-	FIELD be RECORD_CAT Lgm;
+	FIELD ba cookie Lgm;
+	FIELD bb map Lwr;
+	FIELD bc shears Lbl;
+	FIELD bd record_13 Lgm;
+	FIELD be record_cat Lgm;
+	FIELD bf id I
 	FIELD bg maxStackSize I
-	FIELD d SHOVEL_IRON Lgm;
-	FIELD e PICKAXE_IRON Lgm;
-	FIELD f HATCHET_IRON Lgm;
-	FIELD g FLINT_AND_STEEL Lgm;
-	FIELD h APPLE Lgm;
-	FIELD i BOW Lgm;
-	FIELD j ARROW Lgm;
-	FIELD k COAL Lgm;
-	FIELD l EMERALD Lgm;
-	FIELD m INGOT_IRON Lgm;
-	FIELD n INGOT_GOLD Lgm;
-	FIELD o SWORD_IRON Lgm;
-	FIELD p SWORD_WOOD Lgm;
-	FIELD q SHOVEL_WOOD Lgm;
-	FIELD r PICKAXE_WOOD Lgm;
-	FIELD s HATCHET_WOOD Lgm;
-	FIELD t SWORD_STONE Lgm;
-	FIELD u SHOVEL_STONE Lgm;
-	FIELD v PICKAXE_STONE Lgm;
-	FIELD w HATCHET_STONE Lgm;
-	FIELD x SWORD_DIAMOND Lgm;
-	FIELD y SHOVEL_DIAMOND Lgm;
-	FIELD z DIAMOND_PICKAXE Lgm;
+	FIELD bk containerItemType Lgm;
+	FIELD c items [Lgm;
+	FIELD d shovelIron Lgm;
+	FIELD e pickaxeIron Lgm;
+	FIELD f hatchetIron Lgm;
+	FIELD g flintAndSteel Lgm;
+	FIELD h apple Lgm;
+	FIELD i bow Lgm;
+	FIELD j arrow Lgm;
+	FIELD k coal Lgm;
+	FIELD l emerald Lgm;
+	FIELD m ingotIron Lgm;
+	FIELD n ingotGold Lgm;
+	FIELD o swordIron Lgm;
+	FIELD p swordWood Lgm;
+	FIELD q shovelWood Lgm;
+	FIELD r pickxaeWood Lgm;
+	FIELD s hatchetWood Lgm;
+	FIELD t swordStone Lgm;
+	FIELD u shovelStone Lgm;
+	FIELD v pickaxeStone Lgm;
```

Comment:
Overall this mapping is
1. Controversial, needs reviews
2. Needs updating to fit with blayyke's changes